manifests/install.pp

```diff
@@ -45,6 +29,12 @@
     ensure  => $cvmfs_version,
   }
 
+  if $fuse3 {
```

Comment:
I wonder if it would be good option to offer a way to roll back and remove `cvmfs-fuse3`. In that case, if `$fuse3==false` we do a `ensure => absent`.

Reply:
I considered this - I'm a bit nervous about existing installations.

Could go for:

```puppet
Optional[Boolean] $fuse = undef
```

as a default which would do nothing as the default.

Will do so.